### ISSUE TITLE
Question about BERT config used in the paper

### ISSUE THREAD

**Issue by sylee0124** [Kyriection/Random-MoE-as-Dropout#1]
Hi, I'm curious about the config for BERT model used in the paper. Out of 12 layers of BERT which layers use MoE FFN layer?
Also are you planning share training script and configs for BERT/RoBERTa?
